lib/LedgerSMB/Sysconfig.pm

```diff
@@ -163,7 +167,7 @@ def 'force_username_case',
 
 def 'max_post_size',
     section => 'main',
-    default => 1024 * 1024,
+    default => MAX_POST_SIZE_DEFAULT,
```

Comment:
I think we should interpret these defaults as the canonical definition of the value. Meaning we need `## no critic` here.